### PR TITLE
Patch 1

### PR DESCRIPTION
**Pull request by anneqm**
# Changes

Adding missing word to the readme.md file, line 21, "...can used.." should read "..can be used..."
# Submitter Checklist

These are the criteria that every PR should meet, please check them off as you
review them:

- [N/A] Includes [tests](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if functionality changed/added). 
- [y] Includes [docs](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if user facing)
- [y] Commit messages follow  [commit message best practices](https://github.com/tektoncd/community/blob/master/standards.md#commit-messages)

_See [the contribution guide](https://github.com/tektoncd/triggers/blob/master/CONTRIBUTING.md) for more details._

# Release Notes

```
Correcting minor typo for readability.
```

### CODE REVIEW

**Comment by anneqm** [comment]
@vtereso Vincent, as we discussed. Thanks!

File: docs/create-webhook.yaml [hunk 1]
```diff
@@ -29,7 +29,7 @@ spec:
       default: "github.com"
     - name: WebhookEvents
       description: "List of events the webhook will send notifications for"
-      default: "[\"push\",\"pull_request\"]"
```
Comment (dibyom) [inline docs/create-webhook.yaml]: Was this intentional?

**Comment by anneqm** [comment]
@dibyom Yes.. that additional one was unintentional.   Thank you for catching that. Can you remove it?

**Comment by dibyom** [comment]
Closing in favor of in #239

**Pull request closed without merging**